account/index.ts

```diff
@@ -43,11 +39,26 @@ export const fetchActiveAccounts = async (
   });
 };
 
+export const fetchActiveAccounts = async (
```

Comment:
Does this function become obsolete after the seed vault PR https://github.com/secretkeylabs/xverse-core/pull/169?

Reply:
i have to take a look at that PR and get the context, maybe @m-aboelenein can have a quicker reply

Reply:
this is not related to the seed vault yet it only handles storage of the seed phrase